packages/message/src/Granules.ts

```diff
@@ -181,7 +180,7 @@ export const generateGranuleApiRecord = async ({
     published,
     createdAt: workflowStartTime,
     timestamp,
-    updatedAt: updatedAt || timestamp,
+    updatedAt: updatedAt || Date.now(),
```

Comment:
I _think_ this is ok. We're passing the `updatedAt` in when we write the granule so right now it looks like the only place we'll default to Date.now() is in our tests.